nilearn/plotting/find_cuts.py

```diff
@@ -269,3 +273,59 @@ def find_cut_slices(img, direction='z', n_cuts=12, spacing='auto'):
     cut_coords.sort()
 
     return _transform_cut_coords(cut_coords, direction, affine)
+
+
+def find_parcellation_atlas_cut_coords(label_img):
```

Comment:
We should probably add a way to specify the background label (use the same name for the argument as in NiftiLabelsMasker)